nextmv/nextmv/cloud/scenario.py

```diff
@@ -0,0 +1,206 @@
+"""This module contains definitions for scenario tests."""
+
+import itertools
+from dataclasses import dataclass
+from enum import Enum
+from typing import Any, Optional, Union
+
+
+@dataclass
+class ScenarioConfiguration:
+    """
+    Configuration for a scenario.
+
+    You can define multiple values for a single option, which will result in
+    multiple scenarios being created. For example, if you have a configuration
```

Comment:
> You can define multiple values for a single option, which will result in **_multiple scenarios_** being created

I am actually not fully clear on what exactly we refer to as a "scenario". It seems like console UI may have a different understanding than what this comment states. This comment seems to suggest that any fine-granular / atomic / cannot be broken down further unit is a scenario.

I actually agree with that description, but am not sure whether we apply the terminology consistently. :thinking:

Reply:
Good catch, I meant to say "runs" here, not scenario/